src/ert/feature_toggling.py

```diff
@@ -0,0 +1,13 @@
+import os
+import warnings
+
+if (_sched_str := os.environ.get("ERT_FEATURE_SCHEDULER")) not in ["ON", "OFF", None]:
```

Comment:
`1` and `0` are what is commonly used. A reasonable solution is to have `""` mean unset, `"0"` mean false and any other non-zero non-empty value be true. However, it would be reasonable to add other false strings like `"false"` or `"off"`.

Let's not create code that deviates from how the rest of the universe does it. `1` is on.